### ISSUE TITLE
ClientID needs to be changed once a publish occurs on unauthorized topic

### ISSUE THREAD

**Issue by cagdasdoner** [jpmens/mosquitto-auth-plug#292]
Here is the test case :
1. Subscribe to a topic with user who has no access to that topic declared on ACL.
2. Publish to this topic with the same user info.
3. Add topic access to the user.
4. Need to change the ClientID to get able to post to this topic. Otherwise this error occurs : 
1505933469: |-- mosquitto_auth_acl_check(..., 234234234099sdas, cagdas, nest, MOSQ_ACL_WRITE)
1505933469: |-- aclcheck(cagdas, nest, 2) CACHEDAUTH: 12

Basically tested with mosquitto pub/sub console with defining client ids by -i param.
Possible a cache issue but not sure if it is crucial. 
Thanks.

**Comment by jpmens** [comment]
Read the diagnostic. It says `CACHEDAUTH`.

**Comment by cagdasdoner** [comment]
So better answer imho : 
auth_cacheseconds 0
Thanks anyway.
